### ISSUE TITLE
no-multi-spaces is not working properly with object declaration

### ISSUE THREAD

**Issue by mannok** [eslint/eslint#11587]
**Tell us about your environment**

* **ESLint Version:** 5.16.0
* **Node Version:** 10.15.0
* **npm Version:** 6.4.1

**What parser (default, Babel-ESLint, etc.) are you using?** Babel-ESLint

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```json
{
	"root": true,
	"parserOptions": {
		"ecmaVersion": 6,
		"parser": "babel-eslint"
	},
	"env": {
		"node": true,
		"browser": true,
		"es6": true,
		"jquery": true
	},
	"globals": {
	},
	"plugins": [],
	"extends": ["eslint:recommended"],
	"rules": {
		"no-console": ["off"],
		"indent": ["warn", "tab"],
		"semi": ["off", "never"],
		"no-mixed-spaces-and-tabs": ["warn"],
		"no-multi-spaces": ["warn"],
		"no-unused-vars": ["warn"],
		"no-extra-boolean-cast": ["off"]
	}
}
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
var obj = { licID: this.displayingResult.feature.attributes.WAIVERID,				preparedBy: this.userInfo.LOGINUID };
```

```bash
Fix all auto-fixable problems (in VSCode)
```

**What did you expect to happen?**
The code should be warned with no-multiple-space and reformatted.

**What actually happened? Please include the actual, raw output from ESLint.**
Got no warning and error

p.s. Those spaces are tabs. Seems that ESLint doesn't treat tab as space?

**Comment by jgalp** [comment]
Is anyone working on this or should I pick it up?

**Comment by mannok** [comment]
Reopen it please
